firestore/integration_test/Podfile

```diff
@@ -4,14 +4,14 @@ use_frameworks! :linkage => :static
 
 target 'integration_test' do
   platform :ios, '10.0'
-  pod 'FirebaseFirestore', '9.4.0'
-  pod 'Firebase/Auth', '9.5.0'
+  pod 'FirebaseFirestore', '9.6.0'
```

Comment:
Need to add the slashes back in for this file:
Firebase/Firestore

Reply:
Done.